setup.cfg

```diff
@@ -34,6 +34,7 @@ install_requires =
     userpath>=1.4.1
     argcomplete>=1.9.4, <2.0
     packaging>=20.0
+    importlib_metadata; python_version < '3.8'
```

Comment:
No lower bound limit? Let's pick the latest major as min?

Reply:
That's a good idea, but there is a bigger question:  Do we need importlib_metadata in setup.cfg at all?  

Technically pipx does not "need importlib_metadata installed to minimally run".  I included it because it is, in a way, a "dependency", but a strange one because pipx installs it itself only in the shared libs.  It is not used in pipx code other than `venv_metadata_inspector.py`, which doesn't run in pipx's environment but in the venv + shared_lib environments.

Reply:
I would say it’s not needed here, for the exact reason you said. pipx’s runtime dependencies are entirely separate from its share libs, and having importlib-metadata in one place but not the other does not sound wrong to me.

Reply:
Removed it.